src/components/left/main/ChatTags.module.scss

```diff
@@ -0,0 +1,52 @@
+.wrapper {
+  display: flex;
+  gap: 0.25rem;
+}
+
+.tag {
+  flex-shrink: 0;
+  min-width: 1.5rem;
+  height: 0.75rem;
+  border-radius: 0.2375rem;
+  padding: 0 0.2375rem;
+  font-size: 0.6375rem;
+  line-height: 0.75rem;
+  text-align: center;
+  position: relative;
+  text-transform: uppercase;
+
+  &Background {
```

Comment:
And what's the idea behind using a separate element for the background?

Reply:
There are several predefined colors for each peer color. Check them out, maybe you could just use them here.
https://github.com/Ajaxy/telegram-tt/blame/ab11b605a708913f5730d8bdddb0ca34cbf035bd/src/styles/_common.scss#L255

If not, consider using pseudo-element for background